modules/2_mnist/src/mnist.cpp

```diff
@@ -21,9 +21,30 @@ void loadImages(const std::string& filepath,
     CV_CheckEQ(magicNum, 2051, "");
 
     int numImages = readInt(ifs);
+    int numRows = 28;
+    int numCols = 28;
+
 
     // TODO: follow "FILE FORMATS FOR THE MNIST DATABASE" specification
     // at http://yann.lecun.com/exdb/mnist/
+    for (int i = 0; i < numImages; ++i) {
+        Mat img(numRows, numCols, CV_8U);
+        for (int r = 0; r < numRows; ++r) {
+            for (int c = 0; c < numCols; ++c) {
+                unsigned char byte = 0;
+                //ifs.read((char*)&byte, sizeof(byte));
+                //img.push_back(byte);
+            }
+        }
+        std::cout << "test cout_1" << std::endl;
```

Comment:
Please don't debug in the CI

Reply:
I asked about it because we need to check something. Sorry about that

Reply:
no problem :)